docs/version0.46.md

```diff
@@ -77,6 +78,12 @@ When the `jcmd` tool sends a command to a running VM, the command might throw th
 
 The [`-Xtgc:allocation`](xtgc.md#allocation) option prints thread-specific allocation cache (TLH) statistics in addition to the cumulative allocation statistics.
 
+### New `-XX:[+|-]ShareOrphans` option added
+
+When `-Xshareclasses` was specified, only those class loaders that implemented the OpenJ9's public shared classes cache APIs (and its child class loaders) could store classes to the shared classes cache. Classes from custom class loaders that did not implement shared classes cache APIs were not stored to the shared classes cache because the VM did not have their module or class path information. You can enable class sharing from all class loaders, irrespective of whether the class loader implements the OpenJ9's public shared classes cache API, with the `-XX:+ShareOrphans` option.
```

Comment:
We can link "public shared classes cache APIs" to https://eclipse.dev/openj9/docs/api-shrc/.

There are more than one places mentioning the long phrase "class loaders that implemented/do not implement the OpenJ9's public shared classes cache APIs", maybe we can introduce a terms like "SCC-aware class loader" (class loader that implemented the SCC API) and "SCC unaware class loader" (class loader doesn't implemented the SCC API), so that we do not need to repeat the long phase.

Reply:
In the existing documents, we are usually avoiding the abbreviated term of shared classes cache. For consistency sake, I will not be using SCC. But, I will see how we can avoid the multiple instances of the long phrase that you have mentioned. Thanks!

Reply:
I have added the link in the main topic instead of this What's new topic.